docs/send-data/opentelemetry-collector/data-transformations.md

```diff
@@ -27,16 +37,22 @@ processors:
         statements:
           - set(attributes["password"], SHA256(attributes["password"]))
           - set(attributes["password"], Concat(["passwd", attributes["password"]], "="))
-          - replace_pattern(attributes["message"], "password=([0-9A-Za-z]+_)", attributes["password"])
+          - replace_pattern(body, "password=([0-9A-Za-z]+_)", attributes["password"])
```

Comment:
Should we delete the attribute "message" after we are done with it ?

Reply:
good point, pushed a change to use `delete_key`